content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -383,15 +383,15 @@ Possible errors:
 
 #### <inlinecode>P3020</inlinecode>
 
-"The automatic creation of shadow databases is disabled on Azure SQL. Please set up a shadow database using the `shadowDatabaseUrl` datasource attribute.<br />Read the docs page for more details: https://pris.ly/d/migrate-shadow"
+"The automatic creation of shadow databases is disabled on Azure SQL. Please set up a shadow database using the `shadowDatabaseUrl` datasource attribute.<br />Read the docs page for more details: [https://pris.ly/d/migrate-shadow](https://pris.ly/d/migrate-shadow)"
 
 #### <inlinecode>P3021</inlinecode>
 
-"Foreign keys cannot be created on this database. Learn more how to handle this: https://pris.ly/d/migrate-no-foreign-keys"
+"Foreign keys cannot be created on this database. Learn more how to handle this: [https://pris.ly/d/migrate-no-foreign-keys](https://pris.ly/d/migrate-no-foreign-keys)"
 
 #### <inlinecode>P3022</inlinecode>
 
-"Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database. Please read more here how to handle this: https://pris.ly/d/migrate-no-direct-ddl"
+"Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database. Please read more here about how to handle this: [https://pris.ly/d/migrate-no-direct-ddl](https://pris.ly/d/migrate-no-direct-ddl)"
```

Comment:
Is this a) a fix of what the Engine actually outputs, or b) a fix of the Engine grammar directly here?
If b), then this will be not mirrored in the Engine (needs a PR in prisma-engines).

Reply:
This is a fix of Engine grammar directly here, haven't changed anything in the prisma-engines repo yet. 
This still needs to be required in script which generates this.

Reply:
Here's the PR for prisma-engines repo:
- https://github.com/prisma/prisma-engines/pull/2916